src/testsuite/vec_f128_dummy.c

```diff
@@ -29,7 +29,9 @@
 #define __STDC_WANT_IEC_60559_TYPES_EXT__ 1
 #define __STDC_WANT_IEC_60559_FUNCS_EXT__ 1
 #endif
+#ifndef __clang__
```

Comment:
why? maybe add a comment?

Reply:
/* Disable <math.h> for __clang__ because of a bug involving <floatn.h>
   incombination with -mcpu=power9 -mfloat128. This means that ISO
   mandated <math.h> functions __finitef128, __isinff128, etc are not
   available.  */